etc/profile-a-l/bcompare.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for Beyond Compare by Scooter Software
+# Description: directory and file compare utility
+# Disables the network, which only impacts checking for updates.
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include bcompare.local
+# Persistent global definitions
+include globals.local
+
+# In case the user decides to include disable-programs.inc, still allow KDE's Gwenview to view images via right click -> Open With -> Associated Application
+noblacklist ${HOME}/.config/gwenviewrc
+
+include disable-common.inc
```

Comment:
I would suggest that we use the same comment format as in meld.profile and kdiff3.profile: `Uncomment the next line (or put it into your meld.local) if you don't need to compare files in FOO`

```suggestion
# Uncomment the next line (or put it into your meld.local) if you don't need to compare files in disable-common.inc
#include disable-common.inc
```

Reply:
Committed with `meld` replaced by `bcompare`.

Reply:
I meant to use this message for the others (disable-programs.inc, wusc, ..) too.